test/images/sample-apiserver/Makefile

```diff
@@ -24,7 +24,7 @@ export
 # Get without building to populate module cache
 # Then, get with OS/ARCH-specific env to build
 bin:
-	docker run --rm -it -v "${TARGET}:${TARGET}:Z" k8s.gcr.io/build-image/kube-cross:v1.15.5-1 \
+	docker run --rm -i -v "${TARGET}:${TARGET}:Z" k8s.gcr.io/build-image/kube-cross:v1.15.8-legacy-1 \
```

Comment:
was dropping the `t` intentional?

Reply:
this matches what we have in the main branch: https://github.com/kubernetes/kubernetes/blob/master/test/images/sample-apiserver/Makefile#L27

I can revert this part

Reply:
@liggitt reverted